src/motivation.md

```diff
@@ -0,0 +1,48 @@
+# Motivation
+
+The decentralized internet demands privacy-respecting data provenance!
+True data provenance will make data truly portable and allow users to share it with others as they see fit.
+
+### Non-repudiation: TLS is not enough
+
+![](png-diagrams/data_provenance_none.png)
+
+Transport Layer Security (TLS) plays a crucial role in digital security. TLS protects communication against eavesdropping and tampering. It ensures that the data received by the `User` from the `Server` is both **authentic** and **unchanged**. The data's authenticity is verified by the `User` through trusted Certificate Authorities (CAs). Data integrity is maintained by transmitting a cryptographic hash alongside the data, which safeguards against deliberate alterations.
+
+However, this hash does not provide **non-repudiation**, meaning it cannot serve as evidence for the **authenticity and integrity** of the data to third parties (e.g., a service or an app). Because the hash is transmitted with symmetric encryption, the user could modify the data and compute a corresponding hash after the TLS session is finished.
+
+Achieving non-repudiation requires digital signatures implemented with asymmetric, public-key cryptography.
+
+While the concept seems straightforward, enabling servers to sign data is not a part of the TLS protocol. Even if all data were securely signed, naively forwarding all data could expose too much information, compromising the `User`'s privacy. **Privacy** is a vital social good that must be protected.
+
+<!-- Additionally, most `Server`s lack motivation or incentives to sign data because retaining data translates to added value. This strategy creates a lock-in effect, enriching platforms with user data and gaining a competitive edge over competitors. -->
```

Comment:
Can we not include this paragraph. It's too controversial.

Reply:
I was thinking the same. That is why it is commented out.